src/test/mocks/store.ts

```diff
@@ -56,15 +56,13 @@ export const mockStore: ReduxState = {
       OsmosisLP: false,
       OsmosisSwap: false,
       ThorSwap: false,
+      Cowswap: false,
       Pendo: false,
       IdleFinance: false,
       Axelar: false,
       Yat: false,
       WalletConnectToDapps: false,
-      DashboardBreakdown: false,
```

Comment:
idk why these were still in here - IDE complains, but type checker doesn't as it's a mock